Disable SIP?

Does this require you to disable SIP? I cloned the repository, and when I ran `make`: 
`sudo mkdir -p /usr/share/ptSh
mkdir: /usr/share/ptSh: Operation not permitted
make: *** [install] Error 1`

Upon Googling, it looks like this should maybe install to `usr/local/` instead of `usr/share`? Not sure.

--- a/Makefile
+++ b/Makefile
@@ -1,12 +1,12 @@
 install:
-	sudo mkdir -p /usr/share/ptSh
-	sudo cp src/config /usr/share/ptSh/config
-	sudo cp src/success_failure_cmd.sh /usr/share/ptSh/success_failure_cmd.sh
-	sudo cp src/set_aliases.sh /usr/bin/ptSh_set_aliases
-	sudo cp src/ptLs.sh /usr/bin/ptls
-	sudo cp src/ptPwd.sh /usr/bin/ptpwd
-	sudo cp src/ptMkdir.sh /usr/bin/ptmkdir
-	sudo cp src/ptTouch.sh /usr/bin/pttouch
+	sudo mkdir -p /usr/local/share/ptSh
+	sudo cp src/config /usr/local/share/ptSh/config
+	sudo cp src/success_failure_cmd.sh /usr/local/share/ptSh/success_failure_cmd.sh
+	sudo cp src/set_aliases.sh /usr/local/bin/ptSh_set_aliases
+	sudo cp src/ptLs.sh /usr/local/bin/ptls
+	sudo cp src/ptPwd.sh /usr/local/bin/ptpwd
+	sudo cp src/ptMkdir.sh /usr/local/bin/ptmkdir
+	sudo cp src/ptTouch.sh /usr/local/bin/pttouch
 	mkdir -p ~/.config
 	mkdir -p ~/.config/ptSh
 	cp src/config ~/.config/ptSh/config

--- a/src/config
+++ b/src/config
@@ -1,7 +1,7 @@
 # This is ptSh config file
-# You can find default config file in /usr/share/ptSh/config
+# You can find default config file in /usr/local/share/ptSh/config
 # You should not override default config file
-# Path to your config file should be /home/$USER$/.config/ptSh/config
+# Path to your config file should be /home/$USER/.config/ptSh/config
 
 #
 # Prefixes

--- a/src/ptLs.sh
+++ b/src/ptLs.sh
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-source /usr/share/ptSh/config
+source /usr/local/share/ptSh/config
 test -f ~/.config/ptSh/config && source ~/.config/ptSh/config
 
 arg=""

--- a/src/ptMkdir.sh
+++ b/src/ptMkdir.sh
@@ -1,1 +1,1 @@
-/usr/share/ptSh/success_failure_cmd.sh "mkdir" $@
+/usr/local/share/ptSh/success_failure_cmd.sh "mkdir" $@

--- a/src/ptPwd.sh
+++ b/src/ptPwd.sh
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-source /usr/share/ptSh/config
+source /usr/local/share/ptSh/config
 test -f ~/.config/ptSh/config && source ~/.config/ptSh/config
 
 function setval { printf -v "$1" "%s" "$(cat)"; declare -p "$1"; }

--- a/src/ptTouch.sh
+++ b/src/ptTouch.sh
@@ -1,1 +1,1 @@
-/usr/share/ptSh/success_failure_cmd.sh "touch" $@
+/usr/local/share/ptSh/success_failure_cmd.sh "touch" $@

--- a/src/success_failure_cmd.sh
+++ b/src/success_failure_cmd.sh
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-source /usr/share/ptSh/config
+source /usr/local/share/ptSh/config
 test -f ~/.config/ptSh/config && source ~/.config/ptSh/config
 
 ERR=$( ($(echo $1) $(echo ${@:2})) 2>&1 )

